spec/routing/grantees_routing_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe GranteesController, type: :routing do
+  describe "routing" do
+    it "routes to #index" do
+      expect(get: "/grantees").to route_to("grantees#index")
+    end
+
+    it "routes to #new" do
+      expect(get: "/grantees/new").to route_to("grantees#new")
+    end
+
+    it "routes to #show" do
```

Comment:
[suggestion] please get rid of all the tests unrelated to the `show` route, thanks!